src/explore-education-statistics-common/src/styles/_form.scss

```diff
@@ -6,3 +6,7 @@
 .govuk-textarea[disabled] {
   cursor: not-allowed;
 }
+
+.ReactModalPortal .govuk-radios__conditional--hidden {
+  display: none;
+}
```

Comment:
It's because the hidden class relies on the `js-enabled` being present on the root element, the modal is outside the root element so doesn't inherit the class.